packages/integration-sdk-runtime/src/execution/step.ts

```diff
@@ -163,12 +169,27 @@ export function prepareLocalStepCollection<
           ctx,
         ) ?? {});
         const enabledRecord: StepStartStates = {};
+
+        if (dependenciesCache?.enabled) {
+          ctx.logger.info(
+            `Using dependencies cache found at ${dependenciesCache.filepath}`,
```

Comment:
Same feedback about logging.

```suggestion
            { cacheFilePath: dependenciesCache.filePath }, 'Using dependencies cache',
```